client/post-editor/media-modal/image-editor/image-editor-crop.jsx

```diff
@@ -12,13 +12,10 @@ import classNames from 'classnames';
  */
 import Draggable from 'components/draggable';
 import {
-	getImageEditorCropBounds,
 	getImageEditorAspectRatio,
 	getImageEditorTransform,
-	getImageEditorCrop
```

Comment:
Fixed in #b60ba64. However, now the Reset button doesn't work for crop movement/resizing. 
![selection_029](https://cloud.githubusercontent.com/assets/4988512/18966296/f926f1ae-8680-11e6-9e48-1d7c98fdd8ed.jpg). Should we fix this?